storage/namespace.go

```diff
@@ -230,15 +265,33 @@ func (n *dbNamespace) Tick(softDeadline time.Duration) {
 	}
 
 	// Tick through the shards sequentially to avoid parallel data flushes
-	var r tickResult
+	var (
+		r  tickResult
+		l  sync.Mutex
+		wg sync.WaitGroup
+	)
 	perShardDeadline := softDeadline / time.Duration(len(shards))
+	perShardDeadline *= time.Duration(n.tickWorkersConcurrency)
```

Comment:
Why do we need to multiply here? Seems counterintuitive that the deadline is longer with higher concurrency